### PR TITLE
DM-30711: Added two new MotionState enum values for MTDome.

### PR DESCRIPTION
**Pull request by wvreeven**

### CODE REVIEW

File: python/lsst/ts/idl/enums/MTDome.py [hunk 1]
```diff
@@ -81,3 +82,5 @@ class MotionState(enum.IntEnum):
     PARKING = 6
     STOPPED = 7
     STOPPING = 8
+    GO_STATIONARY = 9
+    STATIONARY = 10
```
Comment (r-owen) [inline python/lsst/ts/idl/enums/MTDome.py:86]: How does this differ from STOPPING or STOPPED?

I am guessing that this is a way of saying "stopping gently" vs "stopping abruptly"? If so, the final state is going to be STOPPED in either case. I hope you can avoid the extra STATIONARY value.

Also if you rename the commands as per my suggestion on ts_xml then the I think you want STOPPING and ESTOPPING or EMERGENCY_STOPPING.

**Review by r-owen — CHANGES_REQUESTED** [review]
I think you have one too many states: STATIONARY and STOPPED are redundant. 

(Also if you do rename the goStationary command then please rename the GO_STATIONARY enum value as well).

File: python/lsst/ts/idl/enums/MTDome.py (hunk 1 above)
Comment (r-owen) [reply python/lsst/ts/idl/enums/MTDome.py:86]: Turns out that GO_STATIONARY means "stopping without putting on the brakes" and STOPPING means "stopping and then putting on the brakes" (or possibly "stopping and putting on the brakes at the same time" which sounds brutal). Similarly for the end states. So I suggest clearer names such as: STOPPING, STOPPING_BRAKING, STOPPED, STOPPED_BRAKED (or possibly with connective words, e.g. "STOPPING_THEN_BRAKING" or "STOPPING_AND_BRAKING" (depending on when the brakes are applied) and "STOPPED_AND_BRAKED".

Comment (wvreeven) [reply python/lsst/ts/idl/enums/MTDome.py:86]: I renamed GO_STATIONARY to STOPPING_BRAKING and STATIONARY to STOPPED_BRAKED.

**Review by r-owen — APPROVED** [review]
Looks great

**Pull request merged**